packages/dx-react-grid/docs/reference/sorting-state.md

```diff
@@ -1,6 +1,6 @@
 # SortingState Plugin Reference
 
-A plugin that manages sorting state. It controls the list of columns that are used to sort the passed rows and their sort order.
+A plugin that manages the sorting state. It controls the list of columns that are used to sort the passed rows and their sort order.
```

Comment:
"sort the passed rows and their sort order." This part doesn't make sense. What are you trying to say here?